dali/python/backend_impl.cc

```diff
@@ -439,24 +584,21 @@ void ExposeTensorList(py::module &m) {
         size_t bytes = volume(tensor_shape)*i_shape.size()*info.itemsize;
 
         // Validate the stride
-        ssize_t dim_prod = 1;
-        for (int i = info.strides.size()-1; i >= 0; --i) {
-          DALI_ENFORCE(info.strides[i] == info.itemsize*dim_prod,
-              "Strided data not supported. Detected on dimension " + std::to_string(i));
-          dim_prod *= info.shape[i];
-        }
+        CheckStrides(info.strides, info.shape, info.itemsize, info.strides.size(),
+                     info.shape.size());
```

Comment:
```suggestion
        CheckStrides(info.strides, info.shape, info.itemsize);
```
The function should take care of obtaining the sizes.

Reply:
Done